hangupsbot/plugins/slackrtm.py

```diff
@@ -1208,7 +1210,10 @@ def handle_ho_message(self, event):
             if sync.hotag:
                 fullname = '%s (%s)' % (fullname, sync.hotag)
             try:
-                photo_url = "http:"+self.bot._user_list.get_user(event.user_id).photo_url
+                if self.bot._user_list.get_user(event.user_id).photo_url is None:
```

Comment:
The try/except-block could be removed now. Using [`event.user`](https://github.com/hangoutsbot/hangoutsbot/blob/master/hangupsbot/event.py#L73) to access the user could improve the performace.